src/fsharp/service/service.fsi

```diff
@@ -633,7 +633,11 @@ type public FSharpChecker =
     /// For example, dependent references may have been deleted or created.
     /// <param name="startBackgroundCompileIfAlreadySeen">Start a background compile of the project if a project with the same name has already been seen before.</param>
     /// <param name="userOpName">An optional string used for tracing compiler operations associated with this request.</param>
-    member InvalidateConfiguration: options: FSharpProjectOptions * ?startBackgroundCompileIfAlreadySeen: bool * ?userOpName: string -> unit    
+    member InvalidateConfiguration: options: FSharpProjectOptions * ?startBackgroundCompileIfAlreadySeen: bool * ?userOpName: string -> unit
+    
+    /// Invalidate a project. Clears all caches related to this project, including check files, stale check files, etc.
```

Comment:
Nit: "this project" --> "the project represented by the given FSharpProjectOptions"